packages/orm/src/drivers/Driver.ts

```diff
@@ -25,5 +25,5 @@ export interface Driver {
 
   flushEntities(em: EntityManager, todos: Record<string, Todo>): Promise<void>;
 
-  flushJoinTables(em: EntityManager, joinRows: Record<string, JoinRowTodo>): Promise<void>;
+  flushJoinTables(em: EntityManager, joinRows: Record<string, JoinRowTodo>): Promise<{ resetAfterFlushed?: () => void }>;
```

Comment:
I think with the `JoinRow.resetAfterFlushed` we should be able to leave this as-is